src/Api/Store/StoreResource.php

```diff
@@ -0,0 +1,85 @@
+<?php
+namespace ShoppingFeed\Sdk\Api\Store;
+
+use Jsor\HalClient\HalResource;
+use ShoppingFeed\Sdk\Api\Catalog\InventoryDomain;
+use ShoppingFeed\Sdk\Api\Order\OrderDomain;
+use ShoppingFeed\Sdk\Api\Order\OrderUpdater;
+use ShoppingFeed\Sdk\Resource\AbstractResource;
+
+class StoreResource extends AbstractResource
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return $this->getProperty('name');
+    }
+
+    /**
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->getProperty('id');
+    }
+
+    /**
+     * @return bool
+     */
+    public function isActive()
+    {
+        return $this->getProperty('status') === 'active';
+    }
+
+    /**
+     * @return string
+     */
+    public function getCountryCode()
+    {
+        return $this->getProperty('country');
+    }
+
+    /**
+     * @return InventoryDomain
+     */
+    public function getInventoryManager()
+    {
+        return new InventoryDomain(
+            $this->resource->getFirstLink('inventory')
+        );
+    }
+
+    /**
+     * @return OrderDomain
+     */
+    public function getOrderManager()
+    {
+        return new OrderDomain(
+            $this->resource->getFirstLink('order')
+        );
+    }
+
+    /**
+     * @return OrderUpdater
+     */
+    public function getOrderUpdater()
```

Comment:
This should be a part of OrderApi object, or we will have too much shortcuts on the Store Object